torchbenchmark/models/llama/__init__.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the GNU General Public License version 3.
+
+
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import NLP
+import torch
+from .model import ModelArgs, Transformer
+import torch
+
+class Model(BenchmarkModel):
+    task = NLP.LANGUAGE_MODELING
+    
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+        self.model_args = ModelArgs(vocab_size=32)
+        self.model = Transformer(self.model_args)
+
+
+        if device == "cuda":
+            torch.set_default_device("cuda")
+        
+        self.model.to(torch.device(device))
+        self.example_inputs = (torch.tensor([[1, 1], [1,1]], dtype=torch.int), 1)
```

Comment:
I think we also need to move the input data to the device
```suggestion
        self.example_inputs = (torch.tensor([[1, 1], [1,1]], dtype=torch.int).to(self.device), 1)
```